app/ui/src/java/org/sakaiproject/tool/gradebook/ui/InstructorViewBean.java

```diff
@@ -273,6 +273,8 @@ public void saveScoresWithoutConfirmation() throws StaleObjectModificationExcept
 			}
 		}
 
+        final Set excessiveScores = getGradebookManager().updateStudentGradeRecords(updatedGradeRecords, getGradebook().getGrade_type(), getStudentUid());
```

Comment:
can you fix the spacing/tabs here?

Reply:
sure. will do.

(it would be nice if soft tabs (4) were used everywhere, but I understand that it also makes sense to use the same tab style that's already in the file)